src/app/ApplicationTemplate.Shared/Framework/Startup/StartupBase.cs

```diff
@@ -44,6 +45,10 @@ public void PreInitialize()
 				throw new InvalidOperationException($"You shouldn't call {nameof(PreInitialize)} more than once.");
 			}
 
+#if __ANDROID__ || __IOS__
```

Comment:
You'll need to add cmd no emit comments, otherwise your code will simply not be there once you run the dotnet new command.